sdk/python/symbolchain/nem/TransactionFactory.py

```diff
@@ -26,12 +25,19 @@ def __init__(self, network, type_parsing_rules=None):
 			nem_type_converter,
 			extend_type_parsing_rules(nem_type_converter, type_parsing_rules))
 
+	def _create_and_extend(self, transaction_descriptor):
```

Comment:
any reason to have separate function instead of this in create?

Reply:
just to share similarity with symbol